app/components/PathToDocument.tsx

```diff
@@ -9,38 +9,34 @@ import CollectionIcon from "~/components/CollectionIcon";
 import Flex from "~/components/Flex";
 
 type Props = {
-  result: DocumentPath;
-  document?: Document | null | undefined;
+  documentPath: DocumentPath;
+  currentDocument?: Document | null | undefined;
   collection: Collection | null | undefined;
-  onSuccess?: () => void;
+  href?: string;
+  onClick?: (ev: React.SyntheticEvent) => void;
   style?: React.CSSProperties;
   ref?: (arg0: React.ElementRef<"div"> | null | undefined) => void;
 };
 
 @observer
 class PathToDocument extends React.Component<Props> {
   handleClick = async (ev: React.SyntheticEvent) => {
-    ev.preventDefault();
-    const { document, result, onSuccess } = this.props;
-    if (!document) {
-      return;
-    }
-
-    if (result.type === "document") {
-      await document.move(result.collectionId, result.id);
-    } else {
-      await document.move(result.collectionId);
-    }
-
-    if (onSuccess) {
-      onSuccess();
+    if (this.props.onClick) {
```

Comment:
This handleClick wrapper is doing nothing now, can just pass `props.onClick` directly